backend/src/main/java/ca/gov/dtsstn/cdcp/api/web/v1/model/mapper/ConfirmationCodeModelAssembler.java

```diff
@@ -0,0 +1,34 @@
+package ca.gov.dtsstn.cdcp.api.web.v1.model.mapper;
+
+import org.springframework.stereotype.Component;
+
+import ca.gov.dtsstn.cdcp.api.service.domain.ConfirmationCode;
+import ca.gov.dtsstn.cdcp.api.web.model.AbstractModelAssembler;
+import ca.gov.dtsstn.cdcp.api.web.v1.controller.ConfirmationCodeController;
+import ca.gov.dtsstn.cdcp.api.web.v1.model.ConfirmationCodeModel;
+import org.springframework.data.web.PagedResourcesAssembler;
+
+import org.springframework.util.Assert;
+
+@Component
+public class ConfirmationCodeModelAssembler
+		extends AbstractModelAssembler<ConfirmationCode, ConfirmationCodeModel> {
+
+	private final ConfirmationCodeModelMapper confirmationCodeModelMapper;
+
+	protected ConfirmationCodeModel instantiateModel(ConfirmationCode code) {
+		Assert.notNull(code, "code is required; it must not be null");
+		return confirmationCodeModelMapper.toModel(code);
+	}
+
+	protected ConfirmationCodeModelAssembler(
+			PagedResourcesAssembler<ConfirmationCode> pagedResourcesAssembler,
+			ConfirmationCodeModelMapper confirmationCodeModelMapper) {
+		super(ConfirmationCodeController.class, ConfirmationCodeModel.class,
+				pagedResourcesAssembler);
+		Assert.notNull(confirmationCodeModelMapper,
+				"confirmationCodeModelMapper is required; it must not be null");
+		this.confirmationCodeModelMapper = confirmationCodeModelMapper;
+	}
+
+}
```

Comment:
```suggestion
@Component
public class ConfirmationCodeModelAssembler extends AbstractModelAssembler<ConfirmationCode, ConfirmationCodeModel> {

	private final ConfirmationCodeModelMapper confirmationCodeModelMapper;

	protected ConfirmationCodeModel instantiateModel(ConfirmationCode confirmationCode) {
		Assert.notNull(confirmationCode, "confirmationCode is required; it must not be null");
		return confirmationCodeModelMapper.toModel(confirmationCode);
	}

	protected ConfirmationCodeModelAssembler(
			PagedResourcesAssembler<ConfirmationCode> pagedResourcesAssembler,
			ConfirmationCodeModelMapper confirmationCodeModelMapper) {
		super(ConfirmationCodeController.class, ConfirmationCodeModel.class, pagedResourcesAssembler);
		Assert.notNull(confirmationCodeModelMapper, "confirmationCodeModelMapper is required; it must not be null");
		this.confirmationCodeModelMapper = confirmationCodeModelMapper;
	}

}
```

Reply:
Add some vertical spacing between the blocks of code (fields, methods, etc) to make it more readable, please. 😺

Reply:
done